app/components/channel_list/header/header.tsx

```diff
@@ -55,47 +61,65 @@ const getStyles = makeStyleSheetFromTheme((theme: Theme) => ({
     },
 }));
 
-const ChannelListHeader = ({displayName, iconPad}: Props) => {
+const ChannelListHeader = ({displayName, iconPad, onHeaderPress}: Props) => {
     const theme = useTheme();
+    const isTablet = useIsTablet();
+    const intl = useIntl();
+    const insets = useSafeAreaInsets();
     const serverDisplayName = useServerDisplayName();
     const marginLeft = useSharedValue(iconPad ? 44 : 0);
     const styles = getStyles(theme);
     const animatedStyle = useAnimatedStyle(() => ({
         marginLeft: withTiming(marginLeft.value, {duration: 350}),
     }), []);
 
-    const intl = useIntl();
-
     useEffect(() => {
         marginLeft.value = iconPad ? 44 : 0;
     }, [iconPad]);
 
+    const onPress = useCallback(() => {
+        const renderContent = () => {
+            return (<PlusMenu/>);
+        };
+
+        const closeButtonId = 'close-plus-menu';
+        bottomSheet({
+            closeButtonId,
+            renderContent,
+            snapPoints: [(4 * ITEM_HEIGHT) + insets.bottom, 10],
```

Comment:
Should we consider if the user can create a new channel (and then is one element less if not)?

Reply:
yes, you are absolutely correct.. the approach for the bottom sheet should be different then, I'll make the required changes for this.